icc-x-api/icc-accesslog-x-api.ts

```diff
@@ -110,7 +110,7 @@ export class IccAccesslogXApi extends IccAccesslogApi {
   }
 
   findByHCPartyPatientSecretFKeys(hcPartyId: string, secretFKeys: string): Promise<Array<AccessLog> | any> {
-    return super.findAccessLogsByHCPartyPatientForeignKeys(hcPartyId, secretFKeys).then((accesslogs) => this.decrypt(hcPartyId, accesslogs))
+    return super.findAccessLogsByHCPartyPatientForeignKeysUsingPost(hcPartyId, secretFKeys.split(',')).then((accesslogs) => this.decrypt(hcPartyId, accesslogs))
```

Comment:
No. Some clients will prefer to use the GET request instead of the POST request.  
Create a second method `findByHCPartyPatientSecretFKeysArray(hcPartyId: string, secretFKeys: Array<string>): Promise<Array<AccessLog> | any>` in the extended API, in order to let the choice to the others, in which you will directly pass the array of string. 
Apply this logic to all the changes you made